imap_processing/codice/utils.py

```diff
@@ -104,23 +116,26 @@ def create_dataset(packets) -> xr.Dataset:
     xarray.Dataset
         xarray dataset containing the metadata
     """
+    # TODO: If SHCOARSE is 0, skip the packet
+    # (This problem may fix itself with valid testing data)
+
     metadata_arrays = collections.defaultdict(list)
 
     for packet in packets:
         add_metadata_to_array(packet, metadata_arrays)
 
     # Convert to datetime64 and normalize by launch date
-    epoch_times = [calc_start_time(item) for item in metadata_arrays["SHCOARSE"]]
+    epochs = [calc_start_time(item) for item in metadata_arrays["SHCOARSE"]]
```

Comment:
Move this down into the array creation itself to avoid the singular/plural confusion of variables.